client/src/components/QueryComponent/QueryComponent.js

```diff
@@ -170,10 +208,21 @@ class QueryComponent extends Component {
 
   render() {
     const { queries, selectedQuery, selectedQueryKey, showSaveQueryModal, newQueryName,
-      showConfirmDeleteModal, showConfirmOverwriteModal, isOverlayResultsChecked } = this.state;
+      showConfirmDeleteModal, showConfirmOverwriteModal, isOverlayResultsChecked, rowHeight, displayEditor, selectedQueryMultiline } = this.state;
 
     return (
       <>
+        <div className="qc-monaco-layer" onBlur={this.handleEditorBlur} style={{ display: displayEditor ? "block" : "none" }} >
```

Comment:
Just out of curiousity... why are we going with `display` CSS rather than just not rendering the component (e.g. `{displayEditor && <Editor...`)?

Reply:
The problem about using this rendering is that the monaco Editor didMount is inserting our Intelissense multiple times and the dispose method is not working properly